lmp/preload-app-images

```diff
@@ -68,7 +72,9 @@ if __name__ == '__main__':
     try:
         preload_app_images(args.installed_versions, args.app_shortlist, args.images_root_dir, args.apps_root_dir, args.dst_dir)
     except Exception as exc:
-        logger.warning('Failed to preload Apps images: {}'.format(exc))
+        logger.exception('Failed to preload Apps images: {}'.format(exc))
+        if args.fail_on_error:
```

Comment:
IIUC the `fail_on_error` is False by default, iirc that's what Mike Scott said is preferable from a user perpsective, OE build should succeed regardless of the preload success.